src/main/haskell/kore/src/Kore/Step/Simplification/AndTerms.hs

```diff
@@ -609,9 +612,9 @@ equalInjectiveHeadsAndEquals
                 (map (ExpandedPattern.predicate . fst) children)
                 (map (ExpandedPattern.substitution . fst) children)
         return
-            ( ExpandedPattern
+            ( Predicated
                 { term = give (MetadataTools.symbolOrAliasSorts tools) $
-                    mkApp firstHead (map (ExpandedPattern.term . fst) children)
+                    mkApp firstHead (map (term . fst) children)
```

Comment:
Not sure if you intended to fix everything with the previous commit, but it didn't include all unqualified usages.